powerauth-webflow-authentication-mtoken/pom.xml

```diff
@@ -40,8 +40,8 @@
             <artifactId>spring-ws-security</artifactId>
             <exclusions>
                 <exclusion>
-                    <artifactId>bcprov-jdk15on</artifactId>
                     <groupId>org.bouncycastle</groupId>
+                    <artifactId>bcprov-jdk15on</artifactId>
```

Comment:
Can we add a comment that in a future version of Spring security the artifact ID may change?